src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ORU_R01/LRI_2.0-NG_CBC_Typ_Message-expected.json

```diff
@@ -361,9 +351,6 @@
         "performer": [
           {
             "reference": "Organization/c8cd57a5-13a6-54d1-7dcd-d0a6b637c660"
-          },
-          {
-            "reference": "PractitionerRole/fbbaa96a-8a5d-f670-0fcf-838b21d4c6a8"
```

Comment:
Just wondering why are we losing this reference?

Reply:
It's because of incorrect editing, I've fixed this issue, thanks!